free-tunes/ExploreMusic/Repository/ExploreMusicRepository.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ExploreMusicRepository.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/03/28.
+//
+
+import Foundation
+
+typealias FetchMusicResult = Result<TracksModel, Error>
+
+class ExploreMusicRepository {
```

Comment:
`ExploreMusicRepository` should be conforming to a protocol so that it can be abstracted.

Reply:
I agree with this 👍🏻